templates/homepage-contribute-band.tmpl

```diff
@@ -6,24 +6,20 @@
         <h2>{{ homepage.contribute.title }}</h2>
       </div>
     </div>
-  </div>
-  <div class="contribute-content">
-    <div class="contribute-row">
+    <div class="contribute-content">
       <div class="contribute-box-left">
         <h3>{{ homepage.contribute.heading }}</h3>
         {% for item in homepage.contribute.ways %}
         <ul>
-          <li><i class="fa fa-star" aria-hidden="true"></i>&ensp;{{ item }}</li>
+          <li>{{ item }}</li>
         </ul>
         {% endfor %}
+        <p>{{ homepage.contribute.main }}</p>
+      </div>
+      <div class="contribute-box-right">
+        <img src="/images/bull-builder.svg" alt="Ansible bull builder." />
```

Comment:
Seems slightly too low for me?

![image](https://github.com/ansible-community/community-website/assets/827786/e07220d4-f6c9-40ec-8c13-107dbddf345b)

Shouldn't the bull's feet be aligned with the last line of text?

Reply:
Also it feels odd to not have a link top-right, like the "View all Ansible events" below it or "View more" blogs above it. For consistency should all the bands have one?

Reply:
Thanks I've fixed both of these and it's much better. Needs a media query for responsiveness but I'll do that in a follow up.